src/main/java/com/github/vkorobkov/jfixturescmd/utils/ExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.github.vkorobkov.jfixturescmd.utils;
+
+import com.beust.jcommander.ParameterException;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public final class ExceptionHandler {
+    private ExceptionHandler() {
+    }
+
+    public static void handleException(Exception e) {
+        if (e instanceof ParameterException) {
+            log.error("Failed to parse command line arguments: " + e.getMessage());
+        } else {
+            log.error(e.getMessage());
```

Comment:
Let's print:
* In case of `ProcessorException`: failed to process fixtures: {message}
* In case of `LoaderException`: failed to load fixtures: {message}
* In case of `ParameterException`: failed to parse command line arguments: {message}

For other exceptions(unexpected) we need to we can print the full stack trace?